setup.py

```diff
@@ -21,22 +21,27 @@ def read(fname):
 
 
 if __name__ == '__main__':
-    if "pip" not in __file__:
-        current_dir = os.path.dirname(__file__)
+    current_dir = os.path.dirname(__file__)
+    dest_dir = os.path.join(current_dir, "ttkthemes", "tkimg")
+
+    if (os.path.isdir(os.path.join(current_dir, "ttkthemes")) and
+        not os.path.exists(dest_dir)):
```

Comment:
As far as I can see, the second part of this if-requirement can safely be removed, and I think it should be. I want the files to be copied *again* every time setup is being run, in case for some reason the same copy of the repository is used on two different operating systems or environments.
Below, in line 40, the existence of the folder is checked already, and if it exists the files are removed for exactly the reason above. `setup.py` should run fine, no matter what the environment or history, which is why the admittedly hacky solution of before should be removed.

Reply:
Have you tried it without it? You can always remove it after merging the pull request.

Reply:
I'll change it after amending the previous commit message for one more time...

Reply:
changed